torch_xla/runtime.py

```diff
@@ -244,9 +244,11 @@ def use_spmd():
 
   if torch_xla._XLAC._xla_get_spmd_config_is_locked(
   ) and not xu.check_env_flag("XLA_USE_SPMD"):
-    raise RuntimeError(
+    warnings.warn(
+        "Replicating tensors already initialized on non-virtual XLA device for SPMD. "
```

Comment:
nit, it is better to call out Pytorch/XLA will handle the data movement for the user but it will be slower. Otherwise this warning sounds like the result is incorrect which scare users.

Reply:
Updated the comment!